shared/js/textChatStore.js

```diff
@@ -148,13 +155,24 @@ loop.store.TextChatStore = (function() {
     },
 
     /**
-     * Handles sending of a chat message.
+     * Handles sending of a chat message.  Adds a name (XXX yuck, this
+     * shouldn't be so side-effecty)
```

Comment:
What do you mean by side-effecty? Because it inserts the `displayName`? We could just have the source that's dispatching `sendTextChatMessage` insert the display name in `textChatView` instead?

It doesn't seem wrong for an action handler to do something like adding a name. Isn't the purpose of a `send` action to cause a side-effect of storing data?